static/js/analytics.js

```diff
@@ -265,27 +267,26 @@ const renderChartsFromSingleUserJson = (json) => {
   renderTaggedFriendsGraphFromJson(json);
 };
 
-const updateDatePicker = (startDate, endDate) => {
+const updateDatePicker = (startDateTimestamp, endDateTimestamp) => {
+  const startDate = new Date(startDateTimestamp).toISOString().split("T")[0];
+  const endDate = new Date(endDateTimestamp).toISOString().split("T")[0];
   document.getElementById("start-date").value = startDate;
   document.getElementById("end-date").value = endDate;
 };
 
 const getDateIntervalAndUpdateDatePicker = () => {
   const startDateField = document.getElementById("start-date").value;
   const endDateField = document.getElementById("end-date").value;
-  const monthsAgo = 3;
-  let endDate = endDateField ? new Date(endDateField) : new Date();
-  let startDate = startDateField
-    ? new Date(startDateField)
-    : new Date(new Date().setMonth(endDate.getMonth() - monthsAgo));
-
-  endDate.setDate(endDate.getDate() + 1); // Handles the last day
-  startDate = startDate.toISOString().split("T")[0];
-  endDate = endDate.toISOString().split("T")[0];
-  const startDateTimestamp = Date.parse(startDate) / 1000;
-  const endDateTimestamp = Date.parse(endDate) / 1000;
-
-  updateDatePicker(startDate, endDate);
+  const currentDate = new Date();
+  let endDateTimestamp = endDateField
```

Comment:
I'd avoid using the same name of the arg.

Reply:
Also this could be a const

Reply:
Done